core/mod/ALU.mod

```diff
@@ -135,7 +135,11 @@ VERBATIM {
     INFOCAST;
     Info* info = (Info*)hoc_Emalloc(sizeof(Info)); hoc_malchk();
     info->psize_ = 10;
+#ifdef NRN_MECHANISM_DATA_IS_SOA
+    info->ptrs_ = new handle_to_double[info->psize_]; hoc_malchk();
+#else
     info->ptrs_ = (handle_to_double*)hoc_Ecalloc(info->psize_, sizeof(handle_to_double)); hoc_malchk();
```

Comment:
@1uc : could you remind me why we need to use NEURON’s handle_to_double? I.e. when I saw this in the morning, I wanted to say this mod file shouldn’t use neurons api for memory management.